src/commands/react-native/README.md

```diff
@@ -91,8 +91,100 @@ The only repository URLs supported are the ones whose host contains: `github`, `
 | GitHub / GitLab  | https://\<repository-url\>/blob/\<commit-hash\>/\<tracked-file-path\>#L\<line\> |
 | Bitbucket | https://\<repository-url\>/src/\<commit-hash\>/\<tracked-file-path\>#lines-\<line\>  |
 
+
+### `xcode`
```

Comment:
I'm wondering if this section should be at the top of the readme?
It'd be wasteful for users that just use `xcode` to waste time reading the "manual" approach above and only realise later (or miss?) that there's an easier 'xcode' setup.

Or alternatively a table of content at the start could be added so users jump straight to the correct instructions.

WDYT?

Reply:
I'm ok to do this change, but to reassure you really few people are going to be starting their sourcemaps integration by reading this README.

For a bit of context, debug symbols integration for React Native is hard as you need to do:
- iOS Dsym
- Android mapping files
- JS sourcemaps on iOS builds (this script)
- JS sourcemaps on Android builds

Nearly all other error tracking tools have a wizard that installs everything for you and automates the upload, so I think most RN devs don't expect to upload their sourcemaps manually.

On the docs website, we are going to have 2 sections:

**Automated setup with `datadog-react-native-wizard`**
This section will explain how to run a wizard that will setup all of this. There won't be any mention of `datadog-ci` here and we expect that most of the users will only use this part

**Manual setup**
This section will describe step by step what the wizard does and how you can customize each one to suit your needs best. Here we'll introduce the `react-native xcode` command and mention that it uses `react-native upload` under the hood.

So I believe very few devs will start by looking at what our CI tool has to offer without getting the bigger picture on the docs website :)

Reply:
Cool, maybe we'll also be able to add a link to the 'official doc' in the readme!